linera-core/benches/client_benchmarks.rs

```diff
@@ -98,20 +95,14 @@ fn criterion_benchmark<M: Measurement + 'static>(c: &mut Criterion<M>) {
     });
 }
 
-fn create_recorder() -> BenchRecorder {
-    let recorder = BenchRecorder::default();
-    recorder.clone().install().unwrap();
-    recorder
-}
-
 criterion_group!(
     name = benches;
     config = Criterion::default()
         .measurement_time(Duration::from_secs(40))
-        .with_measurement(BenchRecorderMeasurement::new(create_recorder(), vec![
-            READ_VALUE_COUNTER, WRITE_VALUE_COUNTER,
-            READ_CERTIFICATE_COUNTER, WRITE_CERTIFICATE_COUNTER,
-            LOAD_VIEW_COUNTER, SAVE_VIEW_COUNTER
+        .with_measurement(BenchRecorderMeasurement::new(vec![
+            "read_value", "write_value",
```

Comment:
This list of strings seem coupled with different parts of the codebase making it difficult to maintain.